### ISSUE TITLE
Could the "double underscore" ligature be added please?

### ISSUE THREAD

**Issue by benjaminreid** [MonoLisaFont/feedback#4]
Hey 👋 

First, this font is lovely. One small request though if you think it’s appropriate.

Could a "double underscore" ligature be added? This exact issue was raised with Fira, https://github.com/tonsky/FiraCode/issues/120.

I think Github actually preview this correctly.

```css
.block {
  &__element {
  }
}
```

Thank you!

**Comment by MarcusSterz** [comment]
Thanks for your feedback – we will certainly consider it!

**Comment by okonet** [comment]
Related discussion: https://component-driven.slack.com/archives/CSHAXUV19/p1592393534012800

**Comment by bebraw** [comment]
Hi,

We discussed through the issue as a team and we looked into implementing the ligature. The outcome of our discussion was the in our opinion, the `__`/`___` case works the best the best without a specific ligature since whitespace between the characters gives the character combination legibility by itself.

It sounds like a little counter-intuitive conclusion but in this case it seems like doing nothing is the best option for Mono Lisa.
